pkg/dashboard/store/ddl/migrations.go

```diff
@@ -168,6 +170,14 @@ CREATE TABLE IF NOT EXISTS key_values (
 			name: defaultValueForRepoPerEnv,
 			stmt: `update environments set repo_per_env=false where repo_per_env is null;`,
 		},
+		{
+			name: addKustomizationPerAppToEnvironmentsTable,
+			stmt: `ALTER TABLE environments ADD COLUMN kustomization_per_app TEXT;`,
+		},
+		{
+			name: defaultValueForKustomizationPerApp,
+			stmt: `update environments set kustomization_per_app=false where kustomization_per_app is null;`,
```

Comment:
This is a default value for environments that are already created.
For those envs, set the value to true. Since it is handled that way.

New envs, will have it disabled by default.

In relation to this:
Please display the repoPerEnv and kustomizationPerApp settings on the environment view. It is non editable, but we should show it to the user.

Reply:
![Screenshot 2023-04-17 at 12 59 56](https://user-images.githubusercontent.com/86593861/232465817-80b0f4e4-032e-420d-aa55-4cd14fc5d4ea.png)